src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -393,6 +411,7 @@ func (updated *Config) updateFrom(updateBytes []byte) error {
 		for domain, _ := range wlDomains {
 			updated.ProxiedSites.Cloud = append(updated.ProxiedSites.Cloud, domain)
 		}
+		sort.Strings(updated.ProxiedSites.Cloud)
```

Comment:
For readability and better diffs, I'm sorting the ProxiedSites.cloud list everywhere now.